Validation/CTPPS/python/simu_config/year_2016_cff.py

```diff
@@ -12,11 +12,12 @@
 # geometry
 from Geometry.VeryForwardGeometry.geometryRPFromDD_2017_cfi import * # using 2017 here is OK
 ctppsCompositeESSource.compactViewTag = ctppsGeometryESModule.compactViewTag
+ctppsCompositeESSource.isRun2 = True
```

Comment:
probably safer (here and below)
```suggestion
ctppsCompositeESSource.isRun2 = ctppsGeometryESModule.isRun2
```

Reply:
@perrotta to pick up the flag configured by era?

Reply:
To pick up the very same `isRun2` flag used in `ctppsCompositeESSource`, as it is done right above for `compactViewTag`, and avoid possible mistakes when one has to modify by hand the same thing in several places.
(Minor comment, admittently)